.github/scripts/work-init.sh

```
@@ -64,3 +70,12 @@ examples)
   exit 1
   ;;
 esac
+
+# Restore the toolchain directive if it was present in the original go.work.
+if [[ -n "${ORIG_TOOLCHAIN:-}" ]]; then
+  if ! go work edit -toolchain="$ORIG_TOOLCHAIN"; 
```

Shellcheck (0.11.0)

[error] Did you forget the 'then' for this 'if'?

(SC1049)